components/button-cards/_generic-card/generic-card.scss

```diff
@@ -102,6 +102,9 @@
     .msds-generic-card__media-info {
       padding: 0 16px;
     }
+    .msds-generic-card__media-info {
+      flex-direction: column;
```

Comment:
I might want to include "flex-direction: column;" in the class above as it seems to be included in a duplicate class of ".msds-generic-card__media-info"